std/math.d

```diff
@@ -4010,35 +4022,47 @@ real modf(real x, ref real i) @trusted nothrow @nogc
  *      $(TR $(TD $(PLUSMN)0.0)      $(TD $(PLUSMN)0.0) )
  *      )
  */
-real scalbn(real x, int n) @safe pure nothrow @nogc
+real scalbn(real x, int n) @safe pure nothrow @nogc { return _scalbn(x,n); }
+
+///
+double scalbn(double x, int n) @safe pure nothrow @nogc { return _scalbn(x,n); }
```

Comment:
```
std/math.d(4025:6)[warn]: Public declaration 'scalbn' has no documented example.
std/math.d(4028:8)[warn]: Public declaration 'scalbn' has no documented example.
```
Huh?

Reply:
You need to use `ditto` to group functions.

Reply:
Ahh, thanks!